opm/input/eclipse/EclipseState/Aquifer/Aquancon.hpp

```diff
@@ -99,7 +99,8 @@ namespace Opm {
             bool operator==(const Aquancon& other) const;
             bool active() const;
 
-            const std::vector<Aquancon::AquancCell>& operator[](int aquiferID) const;
+            bool containAquiferConnectoin(int aquiferID) const;
```

Comment:
Thanks for the suggestion. It is a little tricky to me, since the connection are the ones connecting aquifers and reservoirs. But at least I will address the typos here.